nautobot/extras/models/models.py

```diff
@@ -103,7 +103,8 @@ class ConfigContext(BaseModel, ChangeLoggedModel, ConfigContextSchemaValidationM
     regions = models.ManyToManyField(to="dcim.Region", related_name="+", blank=True)
     sites = models.ManyToManyField(to="dcim.Site", related_name="+", blank=True)
     locations = models.ManyToManyField(to="dcim.Location", related_name="+", blank=True)
-    roles = models.ManyToManyField(to="dcim.DeviceRole", related_name="+", blank=True)
+    # TODO(timizuo): Figure out if roles is still meant to be here.
+    roles = models.ManyToManyField(to="extras.Role", related_name="+", blank=True, null=True)
```

Comment:
Should still be needed, else we're removing functionality as far as assigning a ConfigContext to apply to only specific device role(s).